zeppelin/zeppelin.sh

```diff
@@ -20,14 +20,14 @@
 set -euxo pipefail
 
 # Detect dataproc image version from its various names
-if (! test -v DATAPROC_IMAGE_VERSION) && test -v DATAPROC_IMAGE_VERSION; then
+if (! test -v DATAPROC_IMAGE_VERSION) && test -v DATAPROC_VERSION; then
   DATAPROC_IMAGE_VERSION="${DATAPROC_IMAGE_VERSION}"
```

Comment:
This should be `DATAPROC_IMAGE_VERSION="${DATAPROC_VERSION}"`

Reply:
you are correct once again

Reply:
@erickMunz - we need to change the rvalue on line 24 here

Reply:
This is the last unaddressed change I think Erick